.github/workflows/build-pr.yml

```diff
@@ -215,9 +228,25 @@ jobs:
           ref: ${{ needs.get-refs.outputs.ref }}
           token: ${{ secrets.GH_TOKEN }}
 
+      - name: Read Config
+        run: cat .github/config.env >> $GITHUB_ENV
+
+      # `apt-get` brings in `3.6` which is too old to be compatible with Java 17
+      - name: Install Maven
+        run: |
+          mvn_version=3.9.8
+          install_dir="/opt/maven"
+          mkdir ${install_dir}
+          curl --fail --silent --show-error --location https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/${mvn_version}/apache-maven-${mvn_version}-bin.tar.gz | tar --extract --gzip --strip-components=1 --directory ${install_dir}
+          echo "${install_dir}/bin" >> $GITHUB_PATH
+
+      - name: Test Maven
```

Comment:
> nit- could be combined with `Install Maven`

It can't - `$GITHUB_PATH` is just a variable used when creating `step`s. It doesn't affect the current `step`, and I don't know how to force-refresh.